.github/workflows/nightly-release-v3.yml

```
@@ -465,9 +400,76 @@ jobs:
-        else
-          echo "has_commit_errors=false" >> $GITHUB_OUTPUT
-        fi
+          else
+            echo "has_commit_errors=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Create and push git tag
+        id: git_tag
+        if: |
+          (steps.quick_check.outputs.should_continue == 'true' || github.event.inputs.force_release == 'true') &&
+          steps.check_tag.outputs.has_tag == 'false' && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)